GridTracker.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=GridTracker
+Comment=GridTracker
```

Comment:
Consider adding a better comment, for example:
GridTracker listens to traffic from WSJT-X and displays it on a map